redfish/provider/data_source_redfish_bios.go

```diff
@@ -147,5 +162,59 @@ func (g *BiosDatasource) readDatasourceRedfishBios(d models.BiosDatasource) (mod
 	d.ID = types.StringValue(bios.ID)
 	d.Attributes, diags = types.MapValue(types.StringType, attributes)
 
+	bootOptionsList := make([]models.BiosBootOptions, 0)
+	for i := range bootOptions {
+		bootOptionsList = append(bootOptionsList, newBootOption(bootOptions[i]))
+	}
+	d.BootOptions = bootOptionsList
+
 	return d, diags
 }
+
+// newBootOption converts client.BiosBootOptions to models.BiosBootOptions
+func newBootOption(input *redfish.BootOption) models.BiosBootOptions {
+	return models.BiosBootOptions{
+		BootOptionEnabled:   types.BoolValue(input.BootOptionEnabled),
+		BootOptionReference: types.StringValue(input.BootOptionReference),
+		DisplayName:         types.StringValue(input.DisplayName),
+		ID:                  types.StringValue(input.ID),
+		Name:                types.StringValue(input.Name),
+		UefiDevicePath:      types.StringValue(input.UefiDevicePath),
+	}
+}
+
+// BootOptionsSchema is a function that returns the schema for Boot Options
+func BootOptionsSchema() map[string]schema.Attribute {
+	return map[string]schema.Attribute{
+		"boot_option_enabled": schema.BoolAttribute{
+			MarkdownDescription: "Enable or disable the boot device.",
+			Description:         "Enable or disable the boot device.",
```

Comment:
```suggestion
			Description:         "Whether the boot device is enabled or not.",
```
Same for the MarkdownDescription